com.unity.ml-agents/Runtime/DemonstrationRecorder.cs

```diff
@@ -12,19 +12,30 @@ namespace MLAgents
     [AddComponentMenu("ML Agents/Demonstration Recorder", (int)MenuGroup.Default)]
     public class DemonstrationRecorder : MonoBehaviour
     {
+        /// <summary>
+        /// Whether or not to record demonstrations.
+        /// </summary>
         [Tooltip("Whether or not to record demonstrations.")]
         public bool record;
 
-        [Tooltip("Base demonstration file name. Will have numbers appended to make unique.")]
+        /// <summary>
+        /// Base demonstration file name. Saved filenames will have a unique number appended.
+        /// </summary>
+        [Tooltip("Base demonstration file name. Saved filenames will have a unique number appended.")]
```

Comment:
Not all files will have appended numbers will they? I thought we only started adding numbers after the second one?

Reply:
Good call-out, updated.